package.json

```diff
@@ -14,6 +14,9 @@
       "require": "./dist/cjs/index.js"
     }
   },
+  "bin": {
+    "dwn-server": "./dist/esm/src/main.js"
```

Comment:
@shamilovtim, for my learning: the PR talks about "our CI we run Docker which takes a whopping 7 minutes" being the motivation of the PR, how does this reduce the CI time? I would have though you'd need to touch some CICD script/yaml?

Reply:
@thehenrytsai  https://github.com/TBD54566975/web5-js/pull/325

in there i'm doing `node node_modules/@web5/dwn-server/dist/esm/src/main.js` which will be replaced by `npx @web5/dwn-server`